tests/test_e2e_13_mef_eline.py

```diff
@@ -0,0 +1,1117 @@
+        assert data['active'] is True
+
+    """It is returning Response 500, but it should return a failure code"""
+    @pytest.mark.xfail
+    def test_150_patch_creation_time(self):
+        api_url = KYTOS_API + '/mef_eline/v2/evc/'
+        evc1 = self.create_evc(100)
+
+        start = datetime.now()
+        payload = {
+            "creation_time": start.strftime(TIME_FMT)
+        }
+
+        # It sets a new circuit's creation_time
+        response = requests.patch(api_url + evc1, data=json.dumps(payload),
+                                  headers={'Content-type': 'application/json'})
+        assert response.status_code == 400
+
+    def test_155_patch_evc_active(self):
+        api_url = KYTOS_API + '/mef_eline/v2/evc/'
+        evc1 = self.create_evc(100)
+
+        payload = {
+            "active": False,
+        }
+
+        # It sets a new circuit's creation_time
+        response = requests.patch(api_url + evc1, data=json.dumps(payload),
+                                  headers={'Content-type': 'application/json'})
+        assert response.status_code == 400
+
+        # It verifies EVC's data
+        response = requests.get(api_url + evc1)
+        data = response.json()
+        assert data['active'] is True
+
+    """It is returning Response 200, should be 400"""
+    @pytest.mark.xfail
+    def test_160_patch_request_time(self):
+        api_url = KYTOS_API + '/mef_eline/v2/evc/'
+        evc1 = self.create_evc(100)
+
+        start = datetime.now()
+        payload = {
+            "request_time": start.strftime(TIME_FMT)
+        }
+
+        # It sets a new circuit's creation_time
+        response = requests.patch(api_url + evc1, data=json.dumps(payload),
+                                  headers={'Content-type': 'application/json'})
+        assert response.status_code == 400
+
+        # It verifies EVC's data
+        response = requests.get(api_url + evc1)
+        data = response.json()
+        assert data['request_time'] != start.strftime(TIME_FMT)
+
+    """It is returning Response 500, but it should return a failure code"""
+    @pytest.mark.xfail
+    def test_165_patch_current_path(self):
+        api_url = KYTOS_API + '/mef_eline/v2/evc/'
+        payload1 = {
+            "name": "my evc1",
+            "enabled": True,
+            "dynamic_backup_path": True,
+            "uni_a": {
+                "interface_id": "00:00:00:00:00:00:00:01:1",
+            },
+            "uni_z": {
+                "interface_id": "00:00:00:00:00:00:00:02:1"
+            }
+        }
+
+        response = requests.post(api_url, data=json.dumps(payload1),
+                                 headers={'Content-type': 'application/json'})
+        data = response.json()
+        evc1 = data['circuit_id']
+
+        time.sleep(10)
+
+        payload2 = {
+            "current_path": [{
+                "enabled": False
```

Comment:
current path should be a list of links!

Reply:
This is resolved with commit [5024008](https://github.com/amlight/kytos-end-to-end-tests/pull/69/commits/502400832dba601762aebb15ebc10fea26ce483a)